.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: "Climakitae Documentation"
+    url: "https://climakitae.readthedocs.io/en/latest/"
```

Comment:
These needs to point to `climakitaegui` readthedocs

Reply:
There are no guidelines for contributing to `climakitaegui` so I've updated the documentation link, but left the contribution link the same. Should I just remove the contribution link?

Reply:
Nah that sounds like a wise choice.